correct patch by jkim
http://www.openoffice.org/nonav/issues/showattachment.cgi/47084/patch-i80003
will be go into cws obo15.

Index: libxslt/makefile.mk
===================================================================
RCS file: /cvs/external/libxslt/makefile.mk,v
retrieving revision 1.7
diff -u -r1.7 makefile.mk
--- libxslt/makefile.mk	19 Jul 2007 11:21:19 -0000	1.7
+++ libxslt/makefile.mk	24 Jul 2007 20:33:55 -0000
@@ -96,6 +96,10 @@
 BUILD_DIR=$(CONFIGURE_DIR)
 .ENDIF
 
+.IF "$(OS)"=="FREEBSD"
+xslt_LDFLAGS+=-L$(SOLARLIBDIR)
+.ENDIF
+
 OUT2INC=libxslt$/*.h
 
 .IF "$(OS)"=="MACOSX"
